app/models/BannerTests.scala

```diff
@@ -17,11 +17,23 @@ object BannerTemplate extends Enum[BannerTemplate] with CirceEnum[BannerTemplate
   case object GuardianWeeklyBanner extends BannerTemplate
 }
 
+case class BannerContent(
+  heading: Option[String],
+  messageText: String,
+  highlightedText: Option[String],
+  cta: Option[Cta],
+  secondaryCta: Option[Cta]
+)
+
 case class BannerVariant(
```

Comment:
Once migration is complete, `bannerContent` will become mandatory and the deprecated fields removed